sql/createtables.sql

```diff
@@ -0,0 +1,122 @@
+IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'[dbo].[tree]') AND type in (N'U'))
+ALTER TABLE [dbo].[tree] DROP CONSTRAINT IF EXISTS [FK_tree_species1]
+GO
+IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'[dbo].[tree]') AND type in (N'U'))
+ALTER TABLE [dbo].[tree] DROP CONSTRAINT IF EXISTS [FK_tree_species]
+GO
+IF  EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'[dbo].[species]') AND type in (N'U'))
+ALTER TABLE [dbo].[species] DROP CONSTRAINT IF EXISTS [FK_species_speciestype]
+GO
+/****** Object:  Table [dbo].[user]    Script Date: 2023-07-31 1:43:37 PM ******/
```

Comment:
Lets remove the drop statements -- these could be dangerous if the database already exists (we'd lose data in the existing tables).  Add a comment at the top of the file stating that this script should only be used to create the initial database

Reply:
done